### DISCUSSION TITLE
Welcome to better-grass Discussions!

### DISCUSSION THREAD

**Discussion by FungY911** [FungY911/better-grass#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by CODPUD** [comment]
Hello! Do you know any good budget-friendly VPS providers? (with residential IPs to run getgrass)
